packages/calling-stateful-client/src/Converter.ts

```diff
@@ -110,7 +119,9 @@ export function convertSdkCallToDeclarativeCall(call: SdkCall): CallState {
 /**
  * @private
  */
-export function convertSdkIncomingCallToDeclarativeIncomingCall(call: SdkIncomingCall): DeclarativeIncomingCall {
+export function convertSdkIncomingCallToDeclarativeIncomingCall(
+  call: SdkIncomingCall | /* @conditional-compile-remove(teams-call) */ TeamsIncomingCall
```

Comment:
Use `IncomingCallCommon` with the black magic to map it to `IncomingCall` for stable.

Reply:
I'd say add a file StableCommonTypes.ts and have all SDK type mapping in that file.